doc/md/schema-fields.md

```diff
@@ -362,9 +362,14 @@ func (Group) Fields() []ent.Field {
 }
 ```
 
-Here is another example for writing a reusable validator:
+`MaxLen` does count bytes which may result in unexpected behavior for multibyte char collation like Russian, Chinese, Japanese, Vietnamese, smileys, ... Here is another example for writing a reusable validator:
```

Comment:
```suggestion
Here is another example for writing a reusable validator:
```

I'd add the part about `MaxLen` counting bytes to the Built-in Validators section and tell the user to look at this example for a "fix".